.github/workflows/test.yml

```diff
@@ -404,6 +404,7 @@ jobs:
 
   env-test:
     runs-on: ubuntu-latest
+    timeout-minutes: 3
```

Comment:
Do we have a sense for how long this usually takes to run?

Reply:
yeah they take about 1-2mins usually, so added an extra 1 min buffer here

Reply:
sg!